with-apollo/apollo.js

```diff
@@ -0,0 +1,24 @@
+import { ApolloClient, HttpLink, InMemoryCache } from '@apollo/client';
+// import { setContext } from '@apollo/link-context';
+
+const GRAPHQL_API_URL = 'https://www.graphqlhub.com/graphql';
+// const TOKEN = '';
+
+// const asyncAuthLink = setContext(async () => {
+//   return {
+//     headers: {
+//       Authorization: TOKEN,
+//     },
+//   };
+// });
```

Comment:
No commented out code

Reply:
I left this in there because I wanted to leave an example of how you'd use it if the API required a token.

Not sure how else to communicate that while using a simple example.

Reply:
Add a comment explaining why its there

Reply:
Cool, I have instructions in the README but I'll explain it here too.